packages/at_secondary_server/lib/src/verb/handler/monitor_verb_handler.dart

```diff
@@ -215,7 +136,8 @@ class MonitorVerbHandler extends AbstractVerbHandler {
           "skeEncKeyName": atNotification.atMetadata?.skeEncKeyName,
           "skeEncAlgo": atNotification.atMetadata?.skeEncAlgo,
         };
-      await _sendNotificationToClient(notification);
+
+      await _checkAndSend(notification);
     }
```

Comment:
All of the changes above can be summarized as: Use the standard isAuthorized check, remove unnecessary code, some renaming and docs for readability.